Products/Zuul/facades/templatefacade.py

```diff
@@ -268,7 +268,7 @@ def getThresholdDetails(self, uid):
         """
         threshold = self._getObject(uid)
         template = threshold.rrdTemplate()
-        info = IThresholdInfo(threshold)
+        info = IInfo(threshold)
```

Comment:
This was tricky but basically I needed more flexiability because the "schema form builder" thing is additive. Which means I can't get different fields than the parent field without doing janky things like making certain fields hidden. 
It was cleaner to make a new interface type that descended from IInfo and added my fields there.